Resources/public/js/views/ez-usermenuview.js

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-usermenuview', function (Y) {
+    'use strict';
+    /**
+     * Provides the User Menu View class
+     *
+     * @module ez-usermenuview
+     */
+    Y.namespace('eZ');
+
+    var CLASS_HIDDEN = 'is-user-menu-hidden',
+        CLASS_USER_PROFILE = 'ez-user-profile',
+        CLASS_USER_MENU = 'ez-user-menu',
+        SELECTOR_USER_PROFILE = '.' + CLASS_USER_PROFILE;
+
+    /**
+     * The User Menu view
+     *
+     * @namespace eZ
+     * @class UserMenuView
+     * @constructor
+     * @extends eZ.TemplateBasedView
+     */
+    Y.eZ.UserMenuView = Y.Base.create('userMenuView', Y.View, [], {
```

Comment:
it should extend `Y.eZ.View` to get the `active` attribute and the ability to write plugin for it.